tests/_pure_prompt_nixdevshell.test.fish

```diff
@@ -22,14 +23,31 @@ before_each
     set --universal pure_enable_nixdevshell true
     set --erase IN_NIX_SHELL
 
-    _pure_prompt_nixdevshell
+    # when executing the tests in nixos builds, /nix/store paths are in the $PATH variable
+    begin
+        set --local --export PATH /bin/bash
+
+        _pure_prompt_nixdevshell
+    end
 ) $status -eq $SUCCESS
 
 before_each
 @test "_pure_prompt_nixdevshell: show prompt and status inside nix dev shell" (
     set --universal pure_enable_nixdevshell true
-    set --universal pure_symbol_nixdevshell_prefix "󱄅 "
+    set --universal pure_symbol_nixdevshell_prefix "󱄅"
```

Comment:
Might need revert relating to output from [functions/_pure_prompt_nixdevshell.fish](#diff-d05142a3f478293d3dc10fef9e4fcddc707c0a9c6f2daed96de8ee74f9825792R14)
```suggestion
    set --universal pure_symbol_nixdevshell_prefix "󱄅 "
```